src/boot/categories.toml

```diff
@@ -80,6 +80,12 @@ description = """
 Algorithms for making data smaller.\
 """
 
+[computer-vision]
+name = "Computer Vision"
```

Comment:
To be consistent with the other category names, the "v" in "vision" should be lower case.